### DISCUSSION TITLE
Is it possible to sum up only the last/max values of each day

### DISCUSSION THREAD

**Discussion by docholiday** [grafana/grafana#44195] [Dashboards, Panels & Visualizations]
Hi there,

we monitor a message application with Zabbix and check every hour, how many messages are sent until then, starting from 0 each new day. So the number of messages sent on a specific day would be the last(=max) value) of that day.

I would like to sum up those last(max) values, so I could show a Panel with let's say "Messages sent in the last 7 days" or so.
If i set grafana to give the total for the last 7 days, it sums up every value of every hour, which of course is a wrong number.

So how can I set it to only count the last or max value of each day?

**Comment by semaidi** [comment]
Hello,
it depends on the type of data source, in my case I have little to do the functions tab: groupBy(1d, max)

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
